.vib/symfony/goss/vars.yaml

```diff
@@ -0,0 +1,19 @@
+binaries:
+  - php
+  - composer
+  - gosu
```

Comment:
One last comment, could you please remove `gosu` from this list. Yesterday we merged https://github.com/bitnami/containers/pull/30862 removing this check from the catalog after the tool was internally removed.

Besides that, I'll approve the PR once the VIB triggers are also removed.